python/tests/utils.py

```diff
@@ -8,3 +11,25 @@ def new_column():
 def unwrap_devary(devary):
     return ffi.cast('void*', devary.device_ctypes_pointer.value)
 
+
+def get_dtype(dtype):
+    return {
+        np.float64: libgdf.GDF_FLOAT64,
+        np.float32: libgdf.GDF_FLOAT32,
+        np.int64:   libgdf.GDF_INT64,
+        np.int32:   libgdf.GDF_INT32,
+        np.int8:    libgdf.GDF_INT8,
+    }[np.dtype(dtype).type]
+
+
+def gen_rand(dtype, size):
```

Comment:
Does it make sense to seed the random number generator in this module (or somewhere else?) so that the unit test input data is deterministic?